prow/config/labels.md

```diff
@@ -31,10 +31,6 @@
 - [Labels that apply to pingcap/tiflow, only for PRs](#labels-that-apply-to-pingcaptiflow-only-for-prs)
 - [Labels that apply to ti-community-infra/configs, for both issues and PRs](#labels-that-apply-to-ti-community-infraconfigs-for-both-issues-and-prs)
 - [Labels that apply to ti-community-infra/configs, only for issues](#labels-that-apply-to-ti-community-infraconfigs-only-for-issues)
-- [Labels that apply to ti-community-infra/devstats, for both issues and PRs](#labels-that-apply-to-ti-community-infradevstats-for-both-issues-and-prs)
```

Comment:
This was deleted by scripts/update-labels.sh. Not sure if it is correct. @purelind PTAL Thanks.

Reply:
it's ok, the repo is archived.